recipes/buffer-atob-btoa/src/workflow.ts

```diff
@@ -0,0 +1,57 @@
+import type { Edit, Range, SgRoot } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { removeLines } from '@nodejs/codemod-utils/ast-grep/remove-lines';
+
+export default function transform(root: SgRoot<Js>): string | null {
+  const rootNode = root.root()
+  const edits: Edit[] = [];
+  const linesToRemove: Range[] = [];
+
+  const requireStatements = getNodeRequireCalls(root, 'buffer');
+  const importStatements = getNodeImportStatements(root, 'buffer');
+  const atobFunctionCalls = rootNode.findAll({
+    rule: {
+      pattern: 'buffer.atob($ARG)'
+    }
+  });
+
+  // Remove all buffer require statements
+  for (const statement of requireStatements) {
+    linesToRemove.push(statement.range());
+  }
+
+  // Remove all buffer import statements
+  for (const statement of importStatements) {
+    linesToRemove.push(statement.range());
+  }
+
+  // Rewrite atob function calls
+  for (const call of atobFunctionCalls) {
+    const argMatch = call.getMatch("ARG");
+    if (argMatch) {
+      const arg = argMatch.text();
+      const replacement = `Buffer.from(${arg}, 'base64').toString('binary')`;
+      edits.push(call.replace(replacement));
+    }
+  }
+
+  const btoaFunctionCalls = rootNode.findAll({
+    rule: {
+      pattern: 'buffer.btoa($ARG)'
+    }
+  });
+
+  // Rewrite btoa function calls
+  for (const call of btoaFunctionCalls) {
+    const argMatch = call.getMatch("ARG");
+    if (argMatch) {
+      const arg = argMatch.text();
+      const replacement = `Buffer.from(${arg}, 'binary').toString('base64')`;
+      edits.push(call.replace(replacement));
+    }
+  }
```

Comment:
The code to update btoa and atob is the same, the only difference is the replacement string. I think you can follow a similar approach to the one used in this other recipe.

https://github.com/nodejs/userland-migrations/blob/b4c7cfe01bd593f17a98030ceb74849bcb36bc30/recipes/util-print-to-console-log/src/workflow.ts#L22-L39